src/app/chart/sparkline/sparkline.component.ts

```diff
@@ -61,8 +61,9 @@ export class SparklineComponent extends ChartBase implements DoCheck, OnInit {
    */
   ngDoCheck(): void {
     if (!isEqual(this.config, this.prevConfig) || !isEqual(this.chartData, this.prevChartData)) {
+      const dataChanged = isEqual(this.chartData, this.prevChartData);
```

Comment:
Can this line be placed outside the if-statement? That way, isEqual(this.chartData, this.prevChartData)) is only called once.

Reply:
Done.